configuration/scripts/tests/io_suite.ts

```diff
@@ -3,7 +3,7 @@ smoke          gx3    32x1        run1year,histhrly,ionetcdf,iocdf2,short
 smoke          gx3    32x1        run1year,histhrly,iopio2,iocdf5,short
 
 # Need to check with cprnc manually, CICE tools always produce diffs between netcdf and pio
-smoke          gx3     8x2        histall,ionetcdf,iocdf5,run5day
+smoke          gx3     8x2        histall,cmip,ionetcdf,iocdf5,run5day
```

Comment:
I'm not sure this is what we want in the test suite.  histall already turns on all the variables, adding cmip here has little or no impact.  How about if we add a new restart test instead like,

restart        gx3    12x2        cmip,ionetcdf,iocdf2